apps/target_manager.py

```diff
@@ -20,16 +21,25 @@ def create_target(targets_json, compose_apps, ota_lite_tag, git_sha, in_version=
     with open(targets_json) as f:
         data = json.load(f)
         latest_version = -1
+        latest_tag_version = {}
+
         for name, target in data['targets'].items():
-            if target['custom']['targetFormat'] == 'OSTREE':
-                ver = int(target['custom']['version'])
-                latest_version = ver if latest_version < ver else latest_version
+            hwid = target['custom']['hardwareIds'][0]
+            ver = int(target['custom']['version'])
+            latest_version = ver if latest_version < ver else latest_version
+            # create new Targets based on existing Targets that
+            # 1. are OSTREE type (TODO: it's redundant since we have only ostree Targets)
```

Comment:
are you sure we can really state its redundant? A user could choose to put non-ostree targets into this?

Reply:
>A user could choose to put non-ostree targets into this?

How can a user create a non-ostree target? Anyway, it's not a big deal I can remove this comment.

Reply:
I removed the TODO comment.